backend/database_handler/transactions_processor.py

```diff
@@ -297,11 +301,13 @@ def set_transaction_appeal(self, transaction_hash: str, appeal: bool):
         )
         # You can only appeal the transaction if it is in accepted or undetermined state
         # Setting it to false is always allowed
-        if (
-            (not appeal)
-            or (transaction.status == TransactionStatus.ACCEPTED)
-            or (transaction.status == TransactionStatus.UNDETERMINED)
-        ):
+        if appeal:
```

Comment:
```suggestion
        if appeal and ((transaction.status == TransactionStatus.ACCEPTED) or (
                transaction.status == TransactionStatus.UNDETERMINED
            ))
            self.set_transaction_timestamp_appeal(transaction, int(time.time()))
        transaction.appealed = appeal
```
Just avoid to repeat the code transaction.appealed = appeal

Reply:
Would have been nice but the transaction.status check exists to prevent putting it on True when the transaction is in another state. So, I am not going to do this.

Reply:
alright, it is just that the nested ifs are very ugly and hard to read. 
What about this:
```python
if not appeal:
    transaction.appealed = appeal
    self.session.commit()
elif transaction.status in (TransactionStatus.ACCEPTED, TransactionStatus.UNDETERMINED):
    transaction.appealed = appeal
    self.set_transaction_timestamp_appeal(transaction, int(time.time()))

```

Reply:
I like it! Solved in a2c95b5eb112528d7e051a20004fda2d028d475b